package.json

```diff
@@ -1,30 +1,24 @@
 {
-  "name": "use-simple-state",
-  "version": "0.0.30",
-  "description": "A simple, dependency-free state manager for React using hooks.",
-  "scripts": {
-    "build": "webpack --config webpack.config.js",
-    "release": "yarn build && yarn publish"
-  },
-  "main": "lib/main.js",
-  "author": "Josh Jahans",
-  "license": "MIT",
-  "private": false,
-  "keywords": [
-    "react",
-    "state",
-    "simple",
-    "hooks",
-    "redux"
-  ],
-  "peerDependencies": {
-    "react": "^16.7.0"
-  },
-  "devDependencies": {
```

Comment:
Can we get the dev deps back please

Reply:
Can you elaborate why it's needed? I was trying to reduce dependencies and files in the code

Reply:
Because they're necessary to develop the library. Dev dependencies have zero bearing on the final artefact. I would strongly recommend reading up on dev, peer and normal dependencies